examples/platform/silabs/efr32/rs911x/rsi_if.c

```diff
@@ -492,9 +493,11 @@ static void wfx_rsi_do_join(void)
         case WFX_SEC_WPA2:
             connect_security_mode = RSI_WPA_WPA2_MIXED;
             break;
+#if WIFI_ENABLE_SECURITY_WPA3_TRANSITION
         case WFX_SEC_WPA3:
-            connect_security_mode = RSI_WPA3;
+            connect_security_mode = RSI_WPA3_TRANSITION;
```

Comment:
there should another macro for `CHIP_917` wrapping this value and adding `RSI_WPA3_PERSONAL_TRANSITION`

Reply:
```suggestion
#ifdef CHIP_9117
            connect_security_mode = RSI_WPA3_TRANSITION;
#else
            connect_security_mode = RSI_WPA3_PERSONAL_TRANSITION;
#endif /* CHIP_9117 */
```